src/DynamoCoreWpf/Utilities/NodeContextMenuBuilder.cs

```diff
@@ -371,6 +372,8 @@ private static void DismissedAlertsMenuItemOnClick(object sender, RoutedEventArg
             if (!(e.OriginalSource is MenuItem menuItem)) return;
 
             NodeViewModel.ErrorBubble.UndismissMessageCommand.Execute(menuItem.Header);
+
+            Analytics.TrackEvent(Actions.DismissedAlerts, Categories.NodeContextMenuOperations);
```

Comment:
The function name is confusing :) I belive this is actually where user undismiss alerts.